arch/arm/src/stm32h7/hardware/stm32h7x3xx_pwr.h

```diff
@@ -121,16 +121,24 @@
 
 /* Power control register 3 (CR3) */
 
-#define STM32_PWR_CR3_BYPASS        (1 << 0)  /* Bit 0: Power management unit bypass */
-#define STM32_PWR_CR3_LDOEN         (1 << 1)  /* Bit 1: Low drop-out regulator enable */
-#define STM32_PWR_CR3_LDOESCUEN     (1 << 2)  /* Bit 2: Supply configuration update enable */
-                                              /* Bits 3-7: Reserved */
-#define STM32_PWR_CR3_VBE           (1 << 8)  /* Bit 8: VBAT charging enable */
-#define STM32_PWR_CR3_VBRS          (1 << 9)  /* Bit 9: VBAT charging resistor selection */
-                                              /* Bits 10-23: Reserved */
-#define STM32_PWR_CR3_USB33DEN      (1 << 24) /* Bit 24: VDD33USB voltage level detector enable */
-#define STM32_PWR_CR3_USBREGEN      (1 << 25) /* Bit 25: USB regulator enable */
-#define STM32_PWR_CR3_USB33RDY      (1 << 26) /* Bit 26: USB supply ready */
+#define STM32_PWR_CR3_BYPASS                (1 << 0)  /* Bit 0: Power management unit bypass */
+#define STM32_PWR_CR3_LDOEN                 (1 << 1)  /* Bit 1: Low drop-out regulator enable */
+#define STM32_PWR_CR3_LDOESCUEN             (1 << 2)  /* Bit 2: Supply configuration update enable */
+#define STM32_PWR_CR3_SMPSEXTHP             (1 << 3)  /* Bit 3: SMPS step-down converter external power delivery selection */
```

Comment:
Do the added bits need to be #if def-ed based on the existence of a SMPS step-down converter (CONFIG_H7_HAS_SMPS) to ensure we do not use them when reserved?

Reply:
I don't see it necessary. I could create a new config for that (CONFIG_H7_HAS_SMPS doesn't seem to exist) but I've seen other implementations for other MCUs and that check is not implemented. WDYT?

Reply:
>but I've seen other implementations for other MCUs and that check is not implemented

We learned for our mistakes on the STM32

The approach it to splits by chip or control by feature. 

In this case there will be other IC added that have SMPS so add STM32H7_HAS_SMPS to the chip file and that way this can be shared.

Reply:
Done